2021/ahmed2022re/openreview/content.tex

```diff
@@ -0,0 +1,378 @@
+\begin{itemize}
+    \item Ablation study: During the reproducibility study, one of the main difficulties we faced was the excessive amount of time required to conduct a single experiment. In Section \ref{sec:AdditionalResults}, we attempted to address this issue by recommending changes to the default settings supported by our experimental results, while keeping the main claims intact.
+    \item Generalization to larger architectures and datasets: Because we were able to reduce the experimental time, we tested the authors' claim on a larger architecture and dataset to verify that the claim still holds in general.
+ %   \item An approach to reduce model variability: In addition to the solutions proposed by the author, we show another promising batch normalization approach found in recent literature [8] to effectively reduce model variability.
+\end{itemize}
+
+
+\section{Methodology}
+\label{sec:Methodology}
+\subsection{Code}
+The publicly available source code was provided by the original authors in the corresponding \href{https://github.com/ceciliaresearch/nondeterminism_instability}{GitHub repository} . It was written using the Pytorch \cite{paszke2019pytorch} and NumPy libraries with Python 3.7.5. We used the same code and made some adjustments, such as setting the width of the output terminal and downloading the datasets manually. With these minor changes, we were able to run the code according to our experimental environment. The code provides a basic structure that allows numerous architectures to be used with few changes. However, additional code must be added for architectures larger than ResNet-18 \cite{he2016deep}. The code also has command line functionality that allows the user to configure seed values and hyperparameter settings for a specific task. Although the code covers the entire implementation with the exact experimental settings described in the paper, a portion of the code is missing to visualize the results.
+\subsection{Model descriptions}
+We initially chose the ResNet-14 model because it was frequently used for image classification experiments in the original study along with the CIFAR-10 datasets. Due to the large number of trained models required for a single experiment, we did not have time to work with other models except ResNet-34, which we created ourselves to test the generalization of the claim over larger model architectures. Figure \ref{fig:pic0} shows the basic residual block of a ResNet architecture, consisting of two 3x3 convolutional layers followed by batch normalization before activation. On the same basis, Table \ref{table:table1} shows the modular architecture of ResNet-14 and ResNet-34 \cite{he2016deep} without the first 7x7 convolutional layer and the final fully-connected layer. The blocks are shown in parentheses along with the number of output channels, with the multiplier indicating the number of residual blocks in that module. 
+\begin{table}[!htb]
```

Comment:
This table is also breaking compilation, due to use of `bmatrix`. Please debug this region or find an alternative way to display the same information.

Reply:
You can test it by removing this table (and fixing other errors) to get correct compilation.